router/router.go

```diff
@@ -42,6 +44,23 @@ func Initialize(ctx context.Context, ev *env.Env) *gin.Engine {
 	eventGroup.Use(isUserLoggedIn)
 	setupEventGroup(eventGroup)
 
+	userGroup := r.Group("/user")
+	userGroup.Use(isUserLoggedIn)
+	{
+		userGroup.GET("/:user_id", user.GetUserHandler)
+		// TODO: this is for testing purposes only - create user at auth level
+		userGroup.POST("/", user.CreateUserHandler)
+	}
+
+	checkInGroup := r.Group("/checkin")
+	checkInGroup.Use(isUserLoggedIn)
+	{
+		checkInGroup.POST("/", checkin.CreateCheckInHandler)
+		checkInGroup.GET("/:check_in_id", checkin.GetCheckInHandler)
+		// Check-in list for a user
+		checkInGroup.GET("/list", checkin.GetUserCheckInsHandler)
```

Comment:
I don't see any figma for user's checkins list, is it there or adding for future scope?

Reply:
<img width="879" alt="Screenshot 2024-07-07 at 10 48 14 PM" src="https://github.com/DiabolusGX/robin-api/assets/34231951/bc8ebe83-6088-4a51-b550-5cc6a01c598b">
there is a my drive page